src/Hive.IO/Hive.IO/Resources/pv_efficiency.csv

```diff
@@ -1,6 +1,27 @@
+roof-integrated Monocrystalline,0.123076923,0,471.9583206,3834.661355,52.30769231,425,40,These monocrystalline BIPV products come in a variety of tints that make them very suitable for retrofit projects.,roof-integrated,Monocrystalline Roof Panel,Monocrystalline,Blue,16.3,1300x875x6 mm,1.1375,14.32967033,140,12.30769231,3.834661355,471.9583206,123.0769231,52.30769231,425,Megaslate Flair,https://3s-solarplus.ch/wp-content/uploads/2021/07/DB_D_MS-Flair_DE_2021-06.pdf,Switzerland,Cost calculated based on coefficient dervied from ratio of a-Si integrated and a-Si attached (proxy for labor) applied to base monocrystalline cost,Derived from STC on Power Rating,Same as white monocrystalline,mono_facade_blue
+facade-intergrated Monocrystalline,0.171428571,0,905.9419465,5284.661355,52.30769231,425,40,This is a standard facade panel system based on a monocrystalline cell construction. It can be easily applied in new construction and retrofit situations.,facade-intergrated,Monocrystalline Facade Panel,Monocrystalline,None,22.8,1300x875x9 mm,1.1375,20.04395604,195,17.14285714,5.284661355,905.9419465,171.4285714,52.30769231,425,Megaslate Fassade,https://3s-solarplus.ch/wp-content/uploads/2021/07/DB_F_MS-Fassade_DE_2020-11.pdf,Switzerland,Cost calculated based on coefficient dervied from ratio of a-Si integrated and a-Si attached (proxy for labor) applied to base monocrystalline cost plus the difference between the Tinted Poly roof and fa�ade panels (Proxy for fa�ade install premium) ,Derived from STC on Power Rating,Same as white monocrystalline,mono_facade
+roof-intergrated Monocrystalline,0.171428571,0,657.3705179,3834.661355,52.30769231,425,40,This is a standard roof panel system based on a monocrystalline cell construction. It can be easily applied in new construction and retrofit situations.,roof-intergrated,Monocrystalline Roof Panel,Monocrystalline,None,22.8,1300x875x9 mm,1.1375,20.04395604,195,17.14285714,3.834661355,657.3705179,171.4285714,52.30769231,425,Megaslate Fassade,https://3s-solarplus.ch/wp-content/uploads/2021/07/DB_F_MS-Fassade_DE_2020-11.pdf,Switzerland,Cost calculated based on coefficient dervied from ratio of a-Si integrated and a-Si attached (proxy for labor) applied to base monocrystalline cost,Derived from STC on Power Rating,Same as white monocrystalline,mono_roof
```

Comment:
there is a problem with unicode characters: � 
maybe some letter from mac?

Reply:
wow this list is long! @justinfmccarty could you please split this csv into "pv_efficiency.csv" and "bipv_efficiency.csv"? Let us make BIPV its own category in Hive :-)